dev/io.openliberty.mcp.internal/test/io/openliberty/mcp/internal/test/AcceptHeader.java

```diff
@@ -0,0 +1,55 @@
+/*******************************************************************************
+ * Copyright (c) 2025 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package io.openliberty.mcp.internal.test;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+import io.openliberty.mcp.internal.HeaderValidation;
+
+public class AcceptHeader {
```

Comment:
I can't remember whether there's a required naming convention for unit test classes. We should confirm these tests are actually run when doing a gradle build.

Reply:
We checked this, it did run